upstairs/src/lib.rs

```diff
@@ -2425,6 +2497,8 @@ impl Downstairs {
                         self.downstairs_errors.insert(client_id, errors + 1);
 
                         // XXX We don't count read errors here.
```

Comment:
this comment should be in the else block?

Reply:
You just want the `git blame` on this to point to me now!
(I will move it)